web/components/MsoaBoundaries.svelte

```diff
@@ -0,0 +1,175 @@
+<script>
```

Comment:
This component is way too huge, I'll split it up later / as we figure out what we want